numpy/f2py/tests/test_abstract_interface.py

```diff
@@ -3,65 +3,22 @@
 from . import util
 from numpy.f2py import crackfortran
 
+from numpy.testing import assert_
```

Comment:
Minor nit: at some point there was a discussion about removing uses of `assert_` to reduce dependency on `numpy.testing`

Reply:
Excellent point, given the discussion here: https://github.com/scipy/scipy/issues/8583

I agree that we can do without `assert_`, will kill it.